benchopt/utils/dependencies_mixin.py

```diff
@@ -73,56 +71,6 @@ def is_installed(cls, env_name=None, raise_on_not_installed=None,
                 env_name=env_name, raise_on_error=raise_on_not_installed
             ) == 0
 
-    @classmethod
-    def install(cls, env_name=None, force=False):
```

Comment:
While working on enhancing the installation message, I discovered that this method serves the same purpose as [``collect``](https://github.com/benchopt/benchopt/blob/69a62c2b6dd63d944229d0251ec0f4de1e24b93a/benchopt/utils/dependencies_mixin.py#L127) method, yet it is never used.

Perhaps we can delete it?

Reply:
My bad! The CI pointed me to where it is used.